node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodePatcher.java

```diff
@@ -30,19 +31,26 @@
  */
 public class NodePatcher {
 
-    public static final String HARDWARE_FAILURE_DESCRIPTION = "hardwareFailureDescription";
+    private static final String HARDWARE_FAILURE_DESCRIPTION = "hardwareFailureDescription";
+    private static final String WANT_TO_RETIRE = "wantToRetire";
+
     private final NodeFlavors nodeFlavors;
     private final Inspector inspector;
     private final NodeRepository nodeRepository;
 
     private Node node;
+    private List<Node> children;
+    private boolean childrenModified = false;
```

Comment:
Move to `apply` as it only seems to be used there.

Reply:
See above.